handlers.go

```diff
@@ -3687,6 +3687,8 @@ type publicStatsResponse struct {
 	DealsOnChain       int64 `json:"dealsOnChain"`
 	TotalObjectsRef    int64 `json:"totalObjectsRef"`
 	TotalBytesUploaded int64 `json:"totalBytesUploaded"`
+	TotalUsers         int64 `json:"totalUsers"`
```

Comment:
@alvin-reyes please use `sql.NullInt64` for all of these - it is safer if there are null values

Reply:
done on the latest commit.